scripts/lib/CIME/case/check_input_data.py

```diff
@@ -24,7 +24,7 @@ def _download_checksum_file(rundir):
     while protocol is not None:
         protocol, address, user, passwd, chksum_file,_ = inputdata.get_next_server()
         if protocol not in vars(CIME.Servers):
-            logger.warning("Client protocol {} not enabled".format(protocol))
+            logger.info("Client protocol {} not enabled".format(protocol))
```

Comment:
This one felt to me like expected behavior, so I downgraded it from a warning to info. (It is still printed when running the `check_input_data` script because of how cime sets up logging, but if you set up logging to suppress info statements, this no longer appears.)